jest.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "moduleNameMapper": {
+    "single-spa": "<rootDir>/src/single-spa.js"
```

Comment:
Before I added this, jest was looking for the single-spa inside of the lib directory, which is quite frustrating if you're doing a `jest --watch` while modifying the src directory.

I actually have zero idea how jest knew to look in the lib directory, but my guess is that it was checking the package.json for a name and main file.

Either way, I've changed this so we're running the tests against our `src` code instead of `lib` code. This has DX advantages, but maybe is worse if the bundling process is the thing that introduces the bug. But I feel like that's a risk i'm generally okay with.